app/src/main/java/org/pragyan/dalal18/fragment/OrdersFragment.kt

```diff
@@ -230,6 +242,13 @@ class OrdersFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener, OrderIt
         }
     }
 
+    private fun showSwipeToDeleteTour() {
```

Comment:
Did you rebase wrt to master? Why is swipeToDeleteTour() here?

Reply:
Yes. I didn't call the function just didn't use it